github/gql/h.py

```diff
@@ -0,0 +1,182 @@
+import json
+
+from requests import Session
+
+GITHUB_ENDPOINT = 'https://api.github.com/graphql'
+
+
+def make_headers(token):
+    """Make headers dict.
+
+    Args:
```

Comment:
Documenting the functions' parameters is nice, but including the `Args` and `Returns` sections for each function seems a bit excessive. It's easy for these docstrings to get out of date and wrong documentation is usually worse than none.

I think [mypy(https://github.com/python/mypy)] is better suited for this purpose. Since PR CI is a standalone Python 3 project, it would be easy to use it. With mypy, you also get the benefits of static typing. The type checking could be automatically executed by Travis CI.

Reply:
@tomaskrizek, please feel free to submit a pull request. We will be happy to review it.

Reply:
IMO, the way how it is document is quite nice. Omitting the comments would be worse. And as you wrote Tomas, mypy is not currently used in the project. So it is a topic for other PR. There is no need to block this PR with mypy.